TESTING.md

```diff
@@ -71,6 +73,9 @@ Make sure you run lint checker before submitting a pull request. To run lint che
 
 Please ensure that you don't introduce any broken links accidently. For any intentional broken link (e.g. dummy url in unit test), you can add it to [lycheeexclude](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/.lycheeexclude) allow list specifically. 
 
+# Writing Tests
+Conventions and best practices for writing tests can be found in [/src/core/TESTING.md](/src/core/TESTING.md)
```

Comment:
We link to the `main` blob for directly for other links in this file, do you think we should do that for this as well? 

ie, https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/core/TESTING.md

Reply:
I did relative paths here because it works locally too. If i was exploring this readme in my editor, i would be taken to the new doc in the editor itself.